examples/ros-service-call-module/src/JsonSchemaForm/IntegerInput.tsx

```diff
@@ -0,0 +1,64 @@
+import { FC, useState, useEffect } from "react";
+import { TextField } from "@formant/ui-sdk";
+
+interface IIntegerInputProps {
+  jsonSchemaObject: any;
+  currentStateObject: any;
+  property: string;
+  defaultValue: string;
+}
+
+export const IntegerInput: FC<IIntegerInputProps> = ({
+  jsonSchemaObject,
+  currentStateObject,
+  property,
+  defaultValue,
+}) => {
```

Comment:
in this scope, it doesn't _look_ like you use `jsonSchemaObject` beyond just _dereferencing_ the `property` value from it; at the least, consider dereferencing once:

```tsx
const prop = jsonSchemaObject.properties[property];
```

But, alternatively, it might be better to rethink the `IIntegerInputProps` interface to _just_ take the resolved `Property` from `jsonSchemaObject.properties` and avoid passing in larger objects than are needed for the component life-cycle.